src/sql/src/plan/query.rs

```diff
@@ -2870,6 +2870,103 @@ pub fn plan_expr<'a>(
             }
             expr.select().into()
         }
+        Expr::ListSubquery(query) => {
+            if !ecx.allow_subqueries {
+                bail!("{} does not allow subqueries", ecx.name)
```

Comment:
This is either (1) the case also in Postgres or (2) a temporary restriction that needs follow-up work? If (2), can you reference an issue to track this (create one if needed).

Reply:
I'm not aware of it being a temporary restriction.  I've included the `ecx.allow_subqueries` check because the other subquery expressions (`Exists` and `Subquery` above) include it.

Do you know why we would not want to check that condition here?

Reply:
I guess I was tired and didn't appreciate the meaning of `!ecx.allow_subqueries` guard yesterday. Subqueries are conditionally disallowed depending on the context in which they appear.